var/spack/repos/builtin/packages/nccl/package.py

```diff
@@ -17,6 +17,7 @@ class Nccl(MakefilePackage, CudaPackage):
     maintainers("adamjstewart")
     libraries = ["libnccl.so"]
 
+    version("2.21.5-2", sha256="1923596984d85e310b5b6c52b2c72a1b93da57218f2bc5a5c7ac3d59297a3303")
```

Comment:
2.21.5-2 doesn't seem to be available at https://github.com/NVIDIA/nccl/tags, where did you find it?

Reply:
Sorry long weekend, it should be 2.21.5-1. I will fix it and submit another pull request.